lib/server/services/dpl/getAllDplDetectors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { DplDetectorRepository } = require('../../../database/repositories/index.js');
+
+/**
+ * Find and return all DPL detectors
+ *
+ * @return {Promise<SequelizeDplDetector[]>} the DPL detectors list
+ */
+exports.getAllDplDetectors = () => DplDetectorRepository.findAll();
```

Comment:
Please wrap this into a class, as for other services. Low level services may be skipped (and maybe should) but not the high level ones.

Reply:
done